c/ldv-memsafety-bitfields/test-bitfields-2.1_false-valid-free.c

```diff
@@ -7,8 +7,10 @@ struct A {
 	unsigned char a;
 	unsigned char b:2;
 	unsigned char c:2;
+	unsigned char pad1[2];
```

Comment:
Not exactly correct size calculations.
The structure was first `[a*8, b*2, c*2, _*20, d*32, e*8, _*24]` (in bits, not packed, full length=`3*4Byte` ),
then `[a*8, b*2, c*2, _*4, d*32, e*8]` (in bits, packed, full length=`7Byte`)
and now `[a*8, b*2, c*2, _*4, pad1*16, d*32, e*8, pad2*24]` (in bits, packed, full length=`3*4Byte`)